### DISCUSSION TITLE
the support version of the "run-name" in the workflow

### DISCUSSION THREAD

**Discussion by leoujk** [community/community#39937] [Enterprise]
### Select Topic Area

Question

### Body

I saw the document of the github action.
There is a wonderful feature - run-name
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name

However, it seems I cannot use the workflow key.
Error message:  (Line: 6, Col: 1): Unexpected value 'run-name'

I check back the document, and I switch the version to Enterprise 3.5, and the run-name part is not on the page.
Does it mean the feature is not supported?

<img width="830" alt="image" src="https://user-images.githubusercontent.com/87712861/203791229-1fa2411f-e45e-4086-a196-ef389b51f994.png">

Moreover, is it possible to support the feature?

Thanks

**Comment by ossanna16** [comment]
Hi there @leoujk and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by shupoval** [comment]
I'd also be interested in this feature. Unfortunately, GitHub Enterprise Server 3.7.2 does not support this feature. Are there any plans to have it in near future (e.g. 3.8 release?)

**Comment by ChristopherHX** [comment]
docs.github.com sources have GitHub Enterprise Server 3.8 as the plan for release src: https://github.com/github/docs/blob/main/data/features/actions-run-name.yml
_No official statement, but in the same folder are more features paired with planned ghes releases._
